alkos/kernel/include/debug_terminal.hpp

```diff
@@ -0,0 +1,32 @@
+#ifndef KERNEL_INCLUDE_DEBUG_TERMINAL_HPP_
+#define KERNEL_INCLUDE_DEBUG_TERMINAL_HPP_
+
+#include <defines.hpp>
+#include <types.hpp>
+
+/* Should be implement inside architecture code */
+extern "C"
+{
+    size_t DebugTerminalReadLineArch_(char *buffer, size_t buffer_size);
+    void DebugTerminalWriteArch_(const char *buffer);
+}
+
+WRAP_CALL void DebugTerminalWrite(const char *str)
```

Comment:
Should this even be wrapped? Like can't it just be arch dependent. Arhc provides DebugTerminalWrite and that's it